kolibri/core/assets/src/views/StorageNotification.vue

```diff
@@ -121,7 +123,7 @@
         }
       },
       manageChannel() {
-        this.$router.push('/');
+        redirectBrowser(urls['kolibri:kolibri.plugins.device:device_management']());
```

Comment:
Nice! This is a great and straightforward way to manage this that I hadn't thought about 👍

Reply:
There are a couple of potential issues here, the first is purely conceptual but more of a guide:

Core components should not have knowledge about plugins - this is encoding knowledge about a specific plugin into the core Kolibri code.

More practically - this will break of the device plugin has been disabled. Not only will it not redirect anywhere, it will also produce an error because `urls['kolibri:kolibri.plugins.device:device_management']` will be `undefined` so you'll have `TypeError: undefined is not a function` thrown.

The simple thing here to do is an existence check on the property before invoking it - the more systematic approach that addresses the first concern, I would need to fully understand what the intended behaviour is here and why the redirect to `/` was insufficient.

Reply:
I just double checked, and it seems like the `/` redirect is a problem because of the redirect behaviour for learner only devices specifically.

I would suggest just doing the existence check here before invoking the function to avoid any potential errors and we can think about a more systematic approach in the future.

Reply:
Good point, @rtibbles ! I think the original `this.$router.push('/')` might work. The fact that this was a to-do was a lingering item that I had thought about several months ago, to make sure the redirecting was sufficient. But, since there have been changes in the redirect in general, the current option might work? 

There are two scenarios with notifications that require a redirect, and this would be for the superadmin case. The notification could appear at the top of any plugin, and the user would need to navigate to the device/channels page, according the the figma, to manage their content. 

<img width="600" alt="Screenshot 2023-06-06 at 2 40 55 PM" src="https://github.com/learningequality/kolibri/assets/17235236/02905397-9bc8-42f2-80a0-f64098dacc11">